handlers/register.go

```diff
@@ -38,7 +38,7 @@ func Register(mainRouter *mux.Router, svc dao.Service) {
 
 	publicAppRouter.Handle("/{transaction_id}/insolvency/practitioners", HandleCreatePractitionersResource(svc)).Methods(http.MethodPost).Name("createPractitionersResource")
 	publicAppRouter.Handle("/{transaction_id}/insolvency/practitioners", HandleGetPractitionerResources(svc)).Methods(http.MethodGet).Name("getPractitionerResources")
-	publicAppRouter.Handle("/{transaction_id}/insolvency/practitioners/{practitioner_id}", HandleDeletePractitioner(svc)).Methods(http.MethodDelete).Name("deletePractitioner")
+	//publicAppRouter.Handle("/{transaction_id}/insolvency/practitioners/{practitioner_id}", HandleDeletePractitioner(svc)).Methods(http.MethodDelete).Name("deletePractitioner")
```

Comment:
Route to be exposed only if Enablenon600routehandlers is set to true. Please remove comment and move line.

Reply:
I wasn't aware of this so thanks for letting me know and this has been fixed in later commits